server/integration/jgroups/src/main/java/org/infinispan/server/jgroups/JChannelFactory.java

```diff
@@ -247,12 +237,13 @@ public boolean isUnknownForkResponse(ByteBuffer buffer) {
     private void init(TP transport) {
         TransportConfiguration transportConfig = this.configuration.getTransport();
         SocketBinding binding = transportConfig.getSocketBinding();
-        if (binding != null) {
-            SocketFactory factory = transport.getSocketFactory();
-            if (!(factory instanceof ManagedSocketFactory)) {
-                transport.setSocketFactory(new ManagedSocketFactory(factory, binding.getSocketBindings()));
-            }
-        }
+        // TODO Uncomment when WFCORE-1127 is fixed and SocketBindingManager.UnnamedBindingRegistryImpl no longer throws IllegalStateException for unbound sockets
```

Comment:
is there a JIRA for this?
we can create a JIRA and link it to WFCORE-1127 to be notified when it is fixed...